index.bs

```diff
@@ -3072,15 +3095,18 @@ To <dfn>maybe replace event-level report</dfn> given an [=attribution source=]
     return "<code>[=event-level-report-replacement result/drop-new-report-low-priority=]</code>".
 1. [=set/Remove=] |lowestPriorityReport| from the [=event-level report cache=].
 1. Decrement |sourceToAttribute|'s [=attribution source/number of event-level reports=] value by 1.
+1. Let |rateLimitRecord| be the element from [=attribution rate-limit cache=] whose 
```

Comment:
Does this imply that there can only be one such element?

Reply:
Yes, correct.

Reply:
Is it guaranteed that such an element always exists?

Reply:
We should assert that the element exists if it's relevant. If there's a possibility of the element not existing, we should say what to do here.

Reply:
Spec has a section where attribution rate-limit records are regularly removed. Could this be making an assumption that if we get here, it wasn't yet removed (which we haven't tracked the dependencies of)?

Reply:
Attribution rate-limit records are deleted at source time + rate limit window. Since rate-limit window is no shorter than the expiry duration, I think we can assert that such an element always exists IIUC.

Reply:
Doesn't that make an assumption about the relationship between expiry duration and rate limit window?

Reply:
That might be hard to catch if that relationship changes, unless there's a definition for that relationship.

Reply:
Both are constants, we may add a note or assert this relation.

https://wicg.github.io/attribution-reporting-api/#valid-source-expiry-range
https://wicg.github.io/attribution-reporting-api/#attribution-rate-limit-window

Reply:
Added a note for this relation.